src/components/__tests__/Home.test.js

```diff
@@ -0,0 +1,15 @@
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../../redux/configStore';
+import Home from '../pages/Home';
+
+describe('Home page test', () => {
+  it('render test', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <Home />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly write unit tests for several components of the app and some pure functions in Redux code. I can only see test for the Home component.

- [ ] Kindly add integration tests using React Testing Library and perform a mocking data fetching and Redux Store connection.